stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -20,6 +21,8 @@
 from stac_fastapi.types.requests import get_base_url
 from stac_fastapi.types.rfc3339 import DateTimeType
 from stac_fastapi.types.search import BaseSearchPostRequest
+from stac_fastapi.types.stac import (PartialCollection, PartialItem,
```

Comment:
`stac` is imported at the top so in the code you can use `stac.PartialCollection`